decidim-admin/app/controllers/decidim/admin/participatory_space/concerns/has_members.rb

```diff
@@ -30,7 +30,7 @@ def index
 
             def new
               enforce_permission_to :create, :space_member
-              @form = form(MemberForm).from_params({}, privatable_to:)
+              @form = form(MemberForm).from_params({}, participatory_space:)
```

Comment:
I guess you could use the default form context variable, `current_participatory_space`

```suggestion
              @form = form(MemberForm).from_params({})
```

Reply:
Done at 7ccd665e1a7c1df79774a91105b789166d1c38ea (I preferred to do it locally so I didn't miss anything and it only triggered the CI when the full changes where applied)